hooks/pre_commit/20-certs_update.rb

```diff
@@ -13,8 +13,8 @@
   cert_file = param('certs', 'server_cert').value
   key_file  = param('certs', 'server_key').value
 
-  if !app_value(:certs_skip_check) && !cert_file.to_s.empty? &&
-     (app_value(:certs_update_server_ca) || app_value(:certs_update_server))
+  if !app_value(:certs_skip_check) && (!cert_file.to_s.empty? ||
+      !ca_file.to_s.empty? || !key_file.to_s.empty?)
```

Comment:
Would this be easier to understand?
```suggestion
  unless app_value(:certs_skip_check) || cert_file.to_s.empty? || ca_file.to_s.empty? || key_file.to_s.empty?
```

Alternative:
```suggestion
  unless app_value(:certs_skip_check) || [cert_file, ca_file, key_file].any? { |v| v.to_s.empty? }
```

Reply:
I think the logic is not correct. Converting it step by step:

1. `if !app_value(:certs_skip_check) && (!cert_file.to_s.empty? || !ca_file.to_s.empty? || !key_file.to_s.empty?)`

2. `if !app_value(:certs_skip_check) && !(cert_file.to_s.empty? && ca_file.to_s.empty? && key_file.to_s.empty?)`

3. `unless app_value(:certs_skip_check) || (cert_file.to_s.empty? && ca_file.to_s.empty? && key_file.to_s.empty?)`

4. `unless app_value(:certs_skip_check) || [cert_file, ca_file, key_file].all? { |v| v.to_s.empty? }`

I do rather like that and agree it's easier to read. I'll update it in a moment. Thanks for the suggestion!